throttle/test.js

```diff
@@ -38,7 +38,7 @@ describe('throttle', function() {
     throttled.call(11);
     throttled.call(22);
     setTimeout(function() {
-      assert.equal(ctx, 22);
+      assert.equal(ctx, 11);
```

Comment:
Ah I had implemented the old version, thinking it was an additional wrinkle... I kind of liked it better before! It had me scratching my head for a minute until I figured out to execute with the last arguments passed in at the end of the timeout.